include/d/d_cs_game.h

```diff
@@ -354,33 +399,55 @@ class dCsGame_c : public dCs_c {
         /* 0x8C8 */ lytCrawShotCsr_c mCrawShot;
         /* 0x934 */ lytVacuumCsr_c mVacuum;
         /* 0x994 */ u8 field_0x994[0x99C - 0x994]; // seemingly not used
-        /* 0x99C */ bool mCursorTrail;
+        /* 0x99C */ bool mDrawCursorTrailThisFrame;
         /* 0x99D */ u8 field_0x99D[0x9A0 - 0x99D]; // Havent seen this range be set
-        /* 0x9A0 */ bool mCursorActive;
-        /* 0x9A1 */ bool field_0x9A1; // Compares to mCursorActive to change state
-        /* 0x9A2 */ bool field_0x9A2; // A way of signalling Lyt non-normal state.
-        /* 0x9A4 */ CursorType_e mActiveCursorType;
-        /* 0x9A8 */ bool mNextCursor;
-        /* 0x9AC */ CursorType_e mNextCursorType;
     };
 
-    void setNextCursorType(lytItemCursor_c::CursorType_e);
-    void setCursorTypeNoneMaybe();
-    void setCursorTypePointer();
+    void setNextCursorType(CursorType_e);
+    void noneSet();
+    void pointerDefSet();
+    void pointerCatSet();
+
+    void pachinkoSetCharging(bool charging, f32 progress);
+    void dowsingSetParams(const mAng &rot, f32 v1, f32 v2);
+    void dowsingSetUnkWord(UNKWORD v);
+    void vacuumSetNotLocked(f32 v);
+    void vacuumSetLocked();
+    void clawshotsSetLocked(bool locked);
+    void bowStartDrawOrCharge(f32 f1, f32 f2);
+    void bowAimStart();
+    void bowReady();
+
+    void setCursorTypePlayerCam();
+
+    bool isCursorActive() const {
+        return mCursorActive;
+    }
+    void setField0x9A0(u8 val) {
+        mCursorActive = val;
+    }
 
     void offNextCursor() {
-        mCursor.offNextCursor();
+        mNextCursorActive = false;
     }
 
+private:
     static dCsGame_c *sInstance;
+    bool isForcedHidden() const;
 
     /* 0x068 */ m2d::ResAccIf_c mCursorResAcc;
     /* 0x11C */ m2d::ResAccIf_c mMain2DResAcc;
     /* 0x1D0 */ dCursorInterfaceGame_c mCursorIf;
-    /* 0x1EC */ s32 mCursorType;
+    /* 0x1EC */ CursorType_e mCursorType;
     /* 0x1F0 */ dCsGameLyt1_c mLyt1;
     /* 0x290 */ dCsGameLyt2_c mLyt2;
     /* 0x330 */ lytItemCursor_c mCursor;
+    /* 0x9A0 */ bool mCursorActive;
+    /* 0x9A1 */ bool field_0x9A1; // Compares to mCursorActive to change state
+    /* 0x9A2 */ bool field_0x9A2; // A way of signalling Lyt non-normal state.
+    /* 0x9A4 */ CursorType_e mActiveCursorType;
+    /* 0x9A8 */ bool mNextCursorActive;
+    /* 0x9AC */ CursorType_e mNextCursorType;
```

Comment:
I think the offsets here got miscalculated -> from a quick glance at dCsGame_c::execute, mCursorActive is at 0xCD0 (clangd also says this)

Reply:
Good catch, I didn't fix them when moving the properties between classes.